content/blog/2020-06-08-release-notes.md

```diff
@@ -86,9 +86,9 @@ Thanks to everyone for contributing!
 | 12 | Brent Mitton |
 | 3 | Logan Anderson |
 | 3 | Sean MacKay |
-| 2 | jpuri |
-| 1 | DJ |
+| 2 | Jyoti Puri |
+| 1 | DJ Walker |
 
```

Comment:
There's no name convention being used here. The names listed are the names that the developer signed the commit with.

Reply:
Right, I just figured, I might as well name our peers properly while at it.

Reply:
Yeah I'm okay with the change. I'm just noting that this is the name _they_ chose to set in their git config so I don't modify it in release notes. It's not my responsibility to name people. Sometimes people intentionally don't add their full/real name for privacy reasons.